packages/create-sitecore-jss/src/templates/nextjs/package.json

```diff
@@ -49,6 +50,7 @@
     "@types/react-dom": "^18.0.5",
     "@typescript-eslint/eslint-plugin": "^5.49.0",
     "@typescript-eslint/parser": "^5.49.0",
+    "@tanstack/react-query": "^5.17.9",
```

Comment:
Where is this dependency used?

Reply:
it is referenced by the generated file

Reply:
@yavorsk Should you use this [package](https://github.com/Sitecore/jss/commit/fa37c606159b1cc596219c7552cc21541c757c78#diff-1fd4f29b92bf6560c660b979a2968a92e4d1d17cdfb53f2941197d5cfa8d0984R47) instead?

Reply:
It's still referenced [here](https://github.com/Sitecore/jss/pull/1713/files#diff-64d568fafe7888d4cb02d6786de72e81bf1822ed2f87994d3e9a4063dc5d28aaR11)